#!/bin/sh /usr/share/dpatch/dpatch-run
## 328-magic-add-xen.dpatch by Radek Vokal <rvokal@redhat.com>
##
## DP: xen patch, recognizes Xen saved domain

@DPATCH@

diff -Naurp file-4.23.orig/magic/Magdir/linux file-4.23/magic/Magdir/linux
--- file-4.23.orig/magic/Magdir/linux	2008-01-12 17:47:04.000000000 +0000
+++ file-4.23/magic/Magdir/linux	2008-01-12 17:46:42.000000000 +0000
@@ -248,3 +248,9 @@
 >72	string	x		%s]
 >168	string	x		UUID: %s
 
+# Type: Xen, the virtual machine monitor
+# From: Radek Vokal <rvokal@redhat.com>
+0	string		LinuxGuestRecord	Xen saved domain
+#>2	regex		\(name\ [^)]*\)		%s
+>20	search/256	(name			(name
+>>&1	string		x			%s...)
